src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,55 @@
-export const Tabs = () => {};
+/* eslint-disable react/prefer-stateless-function */
+import React from 'react';
+import classNames from 'classnames';
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  tabs: Tab[],
+  selectedTabId: string,
+  onTabSelected: (tab: Tab) => void,
+};
+
+export class Tabs extends React.Component<Props, {}> {
+  render() {
+    const { tabs, selectedTabId, onTabSelected } = this.props;
+
+    const selectedTab = tabs.find(
+      (tab: Tab) => tab.id === selectedTabId,
+    ) || tabs[0];
+
+    return (
+      <div data-cy="TabsComponent">
+        <div className="tabs is-boxed">
+          <ul>
+            {tabs.map((tab: Tab) => {
+              const tabHref = `#${tab.id}`;
+              const active = tab === selectedTab;
```

Comment:
1. name
2. it's better to compare id's, not an objects 
```suggestion
              const isActive = id === selectedTab.id;
```